tests/test_views.py

```diff
@@ -35,6 +35,11 @@ def setUp(self):
     def testServer(self):
         self.assertEqual(404, self.app.get('/').status_code)
 
+    def testCors(self):
+        response = self.app.get('/')
```

Comment:
Since we set CORS on the whole app, this should be sufficient. However, more test cases wouldn't hurt. Eventually we should test individual endpoints, so let's go ahead and implement tests for endpoints that actually do something now. The 404s can wait until we implement them.